Olive.sln

```diff
@@ -154,6 +154,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MSharp.Build", "Utils\MShar
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Olive.Net4x.Compatibility", "Olive.Net4x.Compatibility\Olive.Net4x.Compatibility.csproj", "{53AE4974-CFE9-4FAD-A908-99AE5AEB9D41}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{1B0E7F4A-9C3A-4FF4-AFA7-D5E6390FA4B8}"
+	ProjectSection(SolutionItems) = preProject
+		nuget.exe = nuget.exe
```

Comment:
To push nuget package, `nuget.exe` was needed. however, this file was needed when there was a need to generate nuget package from `nuspec` files.